Added highcharts/3-minimal-charts (#12)
blacklabel/highcharts-fundamentals: highcharts-api/highcharts/3-minimal-charts/README.md

# highcharts/3-minimal-charts

> Create 4 minmal charts that show only series without other chart elements as in the gif below. Use CSS styles in such a way that the charts take up the available space in the parent element. Some chart options are common and some are individual, so use the method learned in the previous task. Also take into account the formatting and behavior of the tooltip. It should all be displayed in small charts and the formatting should be different for pie chart.

Reviewer: Why should formatting be different only for the pie chart? It seems to me that in the gif it is different for each of the charts.
```suggestion
Create 4 minimal charts that show only series without other chart elements, as in the gif below. Use CSS styles in such a way that the charts take up the available space in the parent element. Some chart options are common and some are individual, so use the method learned in the previous task. Also, take into account the formatting and behavior of the tooltip. Everything should be displayed in small charts and the formatting should be different for the pie chart.
```
